Configuration/Skimming/python/Skims_PDWG_cff.py

```diff
@@ -271,6 +271,17 @@
     dataTier = cms.untracked.string('AOD')
     )
 
+from Configuration.Skimming.PDWG_EXODelayedJetMET_cff import *
+EXODelayedJetMETPath = cms.Path(EXODelayedJetMETSkimSequence)
+SKIMStreamEXODelayedJetMET = cms.FilteredStream(
+    responsible = 'PDWG',
+    name = 'EXODelayedJetMET',
+    paths = (EXODelayedJetMETPath),
+    content = skimRawAODContent.outputCommands,
+    selectEvents = cms.untracked.PSet(),
+    dataTier = cms.untracked.string('AOD')
```

Comment:
is this really supposed to have been an `AOD`? @sam7k9621 @davidlange6 @rappoccio

Reply:
Yes, the analysis needs access to the ecal rechits that are only available in AOD

Reply:
Other skims use `USER`, right? @sam7k9621 @davidlange6 @rappoccio please comment